catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/RefreshRegistrySubscriptions.java

```diff
@@ -114,13 +114,18 @@ private void addOrUpdateMetacardsForRemoteRegistries(Set<String> sourceIds)
     private Map<String, Metacard> getRegistryMetacardsMap() throws FederationAdminException {
 
         Map<String, Metacard> registryMetacardsMap = new HashMap<>();
-        List<Metacard> registryMetacards = federationAdminService.getRegistryMetacards();
+        try {
+            List<Metacard> registryMetacards = javax.security.auth.Subject.doAs(subject,
```

Comment:
Talked to @mcalcote about adding it to the class comments as well as the method just be thorough and explicitly clear.